installer/scripts/omsadmin.sh

```diff
@@ -787,6 +804,40 @@ show_workspace_status()
     fi
 }
 
+show_omsagent_proc_limit()
+{
+    local result_limit=
+
+    if [ ! -f $PROC_LIMIT_CONF ]; then
+        log_error "Limits configuration file '$PROC_LIMIT_CONF' missing!"
+        return 1
+    fi
+    local count=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | wc -l`
+
+    if [ "$count" -eq "0" ]; then
+        log_info "Process limit not currently set for $AGENT_USER."
+    elif [ "$count" -eq "1" ]; then
+        result_limit=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | awk '{print $4}'`
+    else
+        log_info "$count $AGENT_USER entries found in $PROC_LIMIT_CONF.  This may yield problems.  Last entry should apply."
+        log_info ">>Last entry presumed to apply<<"
+        local limit_lines=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX"`
+        log_info $limit_lines
+        log_info "End of the $count duplicate entry WARNING Message"
```

Comment:
This is now an info message; change to "End of the $count duplicate entry INFO Message"

Reply:
Yes, of course.